pkgs/development/python-modules/pyschedule/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pulp
+}:
+
+buildPythonPackage rec {
+  pname = "pyschedule";
+  name = "${pname}-${version}";
+  version = "0.2.17";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0mlcmz1cm9yixlvh1d1c6c91j48hhcmbxf8cacqqh7dmgi7pmrs9";
+  };
+
+  propagatedBuildInputs = [ pulp ];
+
+  doCheck = false;
```

Comment:
Why?

Reply:
In other words, add a reason to the source code.

Reply:
Or even better, set to `true` :)